client/data/disputes/hooks.ts

```diff
@@ -30,7 +47,10 @@ export const useDisputeEvidence = () => {
 	return { updateDispute };
 };
 
-export const useDisputes = ( { paged, per_page: perPage } ) =>
+export const useDisputes = ( {
+	paged,
+	per_page: perPage,
+}: DisputesParams ): DisputesResponse =>
```

Comment:
I've done the same changes in my PR but slightly differently. 

There's [whole conversation](https://github.com/Automattic/woocommerce-payments/pull/3491#discussion_r765699241) on this topic where we arrived with idea of inlining definitions. 


IMHO, it's better than creating `DisputesParams` interface as it's more readable when definition is next to actual params.

Reply:
@kalessil would love to hear your thoughts on this?

Reply:
I'd love to have that aligned. Let's inline the return definition.

Reply:
@dpaun1985 in that case you might benefit from waiting for my PR to be merged in the main branch and then syncing.

Reply:
indeed is more readable if we have the definition inline. thx @KarlisJ . I will remove `DisputesParams`